tests/xscconfigprofile_test.go

```diff
@@ -26,17 +54,32 @@ func TestGetConfigurationProfile(t *testing.T) {
 	err = json.Unmarshal(profileFileContent, &configProfileForComparison)
 	assert.NoError(t, err)
 	assert.Equal(t, &configProfileForComparison, configProfile)
+
 }
 
-func createXscMockServerForConfigProfile(t *testing.T) (mockServer *httptest.Server, configProfileService *services.ConfigurationProfileService) {
+func createXscMockServerForConfigProfile(t *testing.T, xrayVersion string) (mockServer *httptest.Server, configProfileService *services.ConfigurationProfileService) {
 	mockServer = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.RequestURI == "/xsc/api/v1/profile/default-test-profile" && r.Method == http.MethodGet {
+		apiUrlPart := "api/v1/"
+		var isXrayAfterXscMigration bool
+		if isXrayAfterXscMigration = xscutils.IsXscXrayInnerService(xrayVersion); isXrayAfterXscMigration {
+			apiUrlPart = ""
+		}
+
+		switch {
+		case strings.Contains(r.RequestURI, "/xsc/"+apiUrlPart+"profile/"+configProfileWithoutRepo) && r.Method == http.MethodGet:
```

Comment:
```suggestion
		case (r.Method == http.MethodGet && strings.Contains(r.RequestURI, "/xsc/"+apiUrlPart+"profile/"+configProfileWithoutRepo)) || (isXrayAfterXscMigration && r.Method == http.MethodPost && strings.Contains(r.RequestURI, "xray/api/v1/xsc/profile_repos")):
```
you can remove the new 'case' below and combine them in the condition since the logic is the same